app/queries/find_pending_upload_failures.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class FindPendingUploadFailures
+  def self.queries
+    [:find_pending_upload_failures]
+  end
+
+  attr_reader :query_service
+  delegate :resources, to: :query_service
+  delegate :resource_factory, to: :query_service
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def find_pending_upload_failures
+    relation = { "pending_uploads" => [], "state" => ["pending"] }
```

Comment:
This should be querying for cases where the `file` uploads have been removed (https://github.com/pulibrary/figgy/blob/master/app/services/file_appender.rb#L175), but the FileSet itself has not had the necessary `FileMetadata` nodes added.

Reply:
https://app.honeybadger.io/projects/53391/faults/57256942 shows that https://github.com/pulibrary/figgy/blob/master/app/models/pending_upload.rb#L13 raises an error, but https://github.com/pulibrary/figgy/blob/master/app/services/file_appender.rb#L178 should remove `PendingUploads` given that https://github.com/pulibrary/figgy/blob/master/app/services/file_appender.rb#L112 captures and logs errors. This query should probably just query for Resources and update the `pending_uploads` attribute when they are missing the `file_name` attribute.

Reply:
`PendingUpload` seems to have been persisted with a random UUID for some years (https://github.com/pulibrary/figgy/blame/dd40b22a78e011b18d942364297d271eca1f605b/app/controllers/base_resource_controller.rb#L34), so I will proceed with trying to just query for cases where these have been persisted with invalid metadata.